command/rootchain/helper/txn.go

```diff
@@ -138,19 +150,25 @@ func FundAccount(account types.Address) (types.Hash, error) {
 	return types.BytesToHash(receipt.TransactionHash.Bytes()), nil
 }
 
-func getJSONRPCClient() *jsonrpc.Client {
-	jrpcClientOnce.Do(func() {
-		ipAddr := ReadRootchainIP()
+func getJSONRPCClient() (*jsonrpc.Client, error) {
+	jrpcClientLock.Lock()
```

Comment:
Is it a lock for jsonRPCClient(is it global variable?)?

Reply:
Yes there was two global variables before
```
	jrpcClientOnce sync.Once
	jsonRPCClient  *jsonrpc.
```
I wanted to get rid of panics in all the functions and because of that I think it is better to initialize `jsonRPCClient` if it is `nil` each time the function is called and not just first time. Lock is used because I suppose intention was to get `jsonRPCClient` from different routines